src/main/java/bio/terra/app/configuration/WebConfig.java

```diff
@@ -34,8 +38,23 @@ public void configurePathMatch(PathMatchConfigurer configurer) {
 
   @Override
   public void addResourceHandlers(ResourceHandlerRegistry registry) {
-    registry
-        .addResourceHandler("/webjars/swagger-ui-dist/**")
-        .addResourceLocations("classpath:/META-INF/resources/webjars/swagger-ui-dist/4.3.0/");
+    Properties properties = new Properties();
+    try (InputStream propsFile =
+        getClass().getClassLoader().getResourceAsStream("swagger-ui.properties")) {
+      properties.load(propsFile);
+      String swaggerUIVersion = properties.getProperty("swagger-ui");
```

Comment:
Looking at this again, there are spring ways to go directly from a custom properties file to a configuration entity that would be better to use. However, we're already doing it this this way elsewhere so this is OK but in the future we'd be better off using the spring APIs
```
@Configuration
@PropertySource("classpath:swagger-ui.properties")
class SwaggerUiConfiguration {
    @Value("${swagger-ui}")
    String version;
```